compiler-rt/lib/tsan/rtl/tsan_dispatch_defs.h

```diff
@@ -56,13 +56,6 @@ extern const dispatch_block_t _dispatch_data_destructor_munmap;
 # define DISPATCH_NOESCAPE
 #endif
 
-#if SANITIZER_APPLE
-# define SANITIZER_WEAK_IMPORT extern "C" __attribute((weak_import))
-#else
```

Comment:
Please move SANITIZER_WEAK_IMPORT in a separate patch

Reply:
https://github.com/llvm/llvm-project/pull/79208

Moved here. I will change the base of this review to be that branch until it is merged.

Reply:
This review has been completed and merged, changing this review back from a draft to a real review and resolving this comment.